test/unit/ngsiv2/expressions/jexlBasedTransformations-test.js

```diff
@@ -248,6 +248,32 @@ const iotAgentConfig = {
                 }
             ],
             explicitAttrs: true
+        },
+        GPS3: {
+            commands: [],
+            type: 'GPS',
+            lazy: [],
+            active: [
+                {
+                    name: 'location',
+                    type: 'geo:json',
+                    expression: "{coordinates: [lon,lat], type: 'Point'}"
+                }
+            ],
+            explicitAttrs: '[ "location" ]'
+        },
+        GPS4: {
+            commands: [],
+            type: 'GPS',
+            lazy: [],
+            active: [
+                {
+                    name: 'location',
+                    type: 'geo:json',
+                    expression: "{coordinates: [lon,lat], type: 'Point'}"
+                }
+            ],
+            explicitAttrs: '[ "loca" + "tion" ]'
```

Comment:
Due to context management, add another test case of a JEXL expression accesing context (for example an static attr, or the value of a measure: 
`explicitAttrs: '[ staticLocation ]'`
`explicitAttrs: '[ activeLocation ]'`

Reply:
add more attrs also (active and staitc to prove they get filtered)

Reply:
and multientity since it's another branch

Reply:
done in https://github.com/telefonicaid/iotagent-node-lib/pull/1180/commits/d9627420dd227a349a8e3fa4ae947d4c0e7592eb